settings_config/celery_config.py

```diff
@@ -24,15 +24,19 @@
 app.autodiscover_tasks(lambda: settings.INSTALLED_APPS)
 # all task
 app.conf.beat_schedule = {
-    # "check_withdrawal": {"task": "check_withdrawal", "schedule": 20.0},
-    # "open_order_on_dydx": {"task": "open_order_on_dydx", "schedule": 20.0},
-    # "close_order_on_dydx": {"task": "close_order_on_dydx", "schedule": 20.0},
+    "check_withdrawal": {"task": "check_withdrawal", "schedule": 20.0},
+    "open_order_on_dydx": {"task": "open_order_on_dydx", "schedule": 20.0},
+    "close_order_on_dydx": {"task": "close_order_on_dydx", "schedule": 20.0},
     "computer_eth_usdc_volatility": {
         "task": "computer_eth_usdc_volatility",
         "schedule": 30.0,
     },    "computer_btc_usdc_volatility": {
         "task": "computer_btc_usdc_volatility",
         "schedule": 30.0,
     },
+    "set_price_floor": {
+        "task": "set_price_floor",
+        "schedule": 30.0,
```

Comment:
30 minutes ?
Should be once a month? At the start?

Reply:
it's 30 second i think

Reply:
when we deploy applications  than we will set it up we still have to discuss  about the calling  time of task

Reply:
oh ok cool

Reply:
Can you add a TODO ?

Reply:
yes sir i have add todo